osm.py

```diff
@@ -38,7 +38,7 @@ def datestring():
     return f'-{datetime.datetime.utcnow().isoformat()}Z'
 
 # Keep this in sync with the format returned by datestring()
-ISO_8601_GLOB = '*-[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9]T[0-9][0-9]:[0-9][0-9]:[0-9][0-9].[0-9]*Z'
+ISO_8601_GLOB = '*-[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9]T[0-9][0-9]:[0-9][0-9]:[0-9][0-9].[0-9][0-9][0-9][0-9][0-9][0-9]Z'
```

Comment:
It's really too bad globs don't support the regex count syntax `{4}` and such.
But then they'd be regexes :-)
I am almost ready to add a new commit that does:
`YEAR = '[0-9]' * 4` and such, but... :-)

Reply:
I feel ya, lol.

I actually _swapped in_ all the `{}` sections, then realized it was glob and not a regex, so I reverted.